package.json

```diff
@@ -3,16 +3,20 @@
     "private": true,
     "scripts": {
         "dev": "next dev",
+        "dev:mocked": "NEXT_PUBLIC_API_MOCKING=enabled next dev",
         "build": "next build",
+        "build:mocked": "NEXT_PUBLIC_API_MOCKING=enabled next build",
         "build:analyze": "ANALYZE=true next build",
         "start": "next start",
+        "start:mocked": "NEXT_PUBLIC_API_MOCKING=enabled next start",
         "lint": "next lint",
         "format:write": "prettier --write .",
         "format:check": "prettier --check .",
         "test": "jest --watch",
         "test:coverage": "jest --coverage",
-        "test:e2e": "npx playwright install --with-deps && npx playwright test",
-        "test:e2e:visual": "npx @percy/cli exec -- npm run test:e2e"
+        "test:e2e": "yarn build:mocked && npx playwright install --with-deps && npx playwright test",
```

Comment:
Sometimes when testing locally, I'll have the app running in development mode, and then run playwright tests on that to see changes in real time. With this change, I'll have to wait for the app to build before the tests run. Would it be possible to just add back that build step in the GitHub Actions pipeline and change it to`yarn build:mocked` instead?